xbmc/addons/binary-addons/AddonDll.cpp

```diff
@@ -317,7 +317,11 @@ void CAddonDll::Destroy()
   m_initialized = false;
 }
 
-ADDON_STATUS CAddonDll::CreateInstance(ADDON_TYPE instanceType, const std::string& instanceID, KODI_HANDLE instance, KODI_HANDLE parentInstance)
+ADDON_STATUS CAddonDll::CreateInstance(ADDON_TYPE instanceType,
+                                       PSEUDO_INSTANCE_CLASS instanceClass,
```

Comment:
Why is this new param prefixed with "PSEUDO" ? I don't really get the meaning for it

Reply:
Is thought to replace after the Game addon use `IAddonInstanceHandler`, as long that have CAddonDll for there, was it only a Pseudo for me :smile:.